packages/components/src/globals/scss/_theme-tokens.scss

```diff
@@ -251,7 +251,7 @@
   /// @type Color
   /// @access public
   /// @group notification
-  $notification-warning-background-color: rgba(#fdd13a, 0.15) !default !global;
+  $notification-warning-background-color: #fff8e2 !default !global;
```

Comment:
Why don't you use the `mix` function to achieve the same thing without having to hard code the hex value?

Reply:
It wasn't giving me the same end result color. We were already hard-coding a hex value here anyways

Reply:
Ahh, I was changing it in the wrong spot. Will update